sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1Test.java

```diff
@@ -628,6 +629,27 @@ public void testSplitQueryFnWithNumSplits() throws Exception {
   }
 
   /**
+   * Tests {@link SplitQueryFn} to make sure that sub-queries are different from the original query
```

Comment:
I think this should be folded into the previous test, and the function `splitQuery` should be changed to generate unequal queries.

Reply:
Done.